config.js

```diff
@@ -53,8 +56,21 @@ module.exports = {
     ],
   },
   logs: {
-    level: 'info', // error, warn, info, verbose, debug, silly
```

Comment:
The lowest (i.e. most verbose) level of log messages to be logged must be configurable.